RELEASE.md

```diff
@@ -49,7 +49,7 @@ For the first release please use the [manual release process](#manual-release-pr
 
 ### What happens if a pull request merged after the automated pull request is created?
 
-We should try to prevent this from happening, but if it does happen, since the pull request runs every wednesday it won't be triggered again and updated. Follow the above [manual release process](#manual-release-process) to include the changes that have been merged.
+We should try to prevent this from happening, but if it does happen, trigger the pull request to update by typing `/prepare-release` once that has completed a 🚀 and 👀 reaction would be added to the comment.
```

Comment:
Do you paste this as a comment on the release PR itself?

Reply:
I think so

Reply:
yes you do, check it out in action here https://github.com/nellyk/one-app/pull/73 and the action triggered here